apps/src/templates/alert.jsx

```diff
@@ -16,19 +18,28 @@ var Alert = React.createClass({
         position: 'absolute',
         zIndex: 1000,
         top: 45,
-        left: 350,
+        left: this.props.leftMargin,
```

Comment:
I don't know that I love this approach, but I think I'm okay with it. The reason this varies between blockly/droplet is that the toolbox is different sizes.